includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -349,20 +351,21 @@ protected function get_row_data( $item ) {
 				);
 				$lesson_completions = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_lesson_completions', $lesson_args, $item ) );
 
-				// Course
+				// Course.
 				$course_id    = get_post_meta( $item->ID, '_lesson_course', true );
 				$course_title = $course_id ? get_the_title( $course_id ) : '';
+				// Taking the ceiling value for the average.
+				$average_completion_days = $lesson_completions > 0 ? ceil( $item->days_to_complete / $lesson_completions ) : __( 'n/a', 'sensei-lms' );
```

Comment:
We're capitalizing the N/A elsewhere, so we should do the same here.

Reply:
Thanks, capitalized in https://github.com/Automattic/sensei/pull/4883/commits/924df553acadf51176832236234004ca471760bf